src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,346 @@
+                    false, false, true, false));
+        } catch (IOException e) {
+            outputDataInConsole("Entered incorrect name of the city");
+        }
+        outputSplitLine();
+        List<Hotel> hotelList = hotelController.findHotelByCity(cityName);
+        if (exitMenu(cityName)) {
+            try {
+                serviceMenu();
+            } catch (IOException e) {
+                outputDataInConsole("Don't loaded Service menu" + e);
+            }
+        }
+        if (hotelList.isEmpty()) {
+            outputDataInConsole("City in " + cityName + " not found.");
+            findHotelByCityMenu();
+        } else {
+            outputDataInConsole("List of hotels in the city found : " + cityName);
+            hotelList.forEach(item -> outputDataInConsole(item.toString()));
+        }
+        hotelList.forEach(item -> outputDataInConsole(item.toString()));
+        return hotelList;
+    }
+
+    private Map<String, List<Room>> findRoomMenu() throws IOException {
+        String city = null;
+        Integer person = null;
+        Integer price = null;
+        Date reservedFrom = null;
+        Date reservedTo = null;
+        try {
+            city = String.valueOf(inputDataFromUser("Enter city name: ", false,
+                    false, true, false));
+            person = (Integer) inputDataFromUser("Enter the number of persons: ", true,
+                    false, false, false);
+            price = (Integer) inputDataFromUser("Enter maximum price rate: ", true,
+                    false, false, false);
+            reservedFrom = (Date) (inputDataFromUser("Enter arrival date in the format dd.mm.yyyy: ",
+                    false, false, false, true));
+            reservedTo = (Date) (inputDataFromUser("Enter the departure date in the format dd.mm.yyyy: ",
+                    false, false, false, true));
+            compareDate(reservedFrom, reservedTo);
+        } catch (IOException e) {
+            outputDataInConsole("Enter arrival date in the format");
+        }
+        Map<String, List<Room>> foundRooms = hotelController.getAvailableRooms(city, person, price,
+                reservedFrom, reservedTo);
+        if (foundRooms.values().isEmpty()) {
+            outputDataInConsole("Room with such parameters not found");
+        } else {
+            outputDataInConsole("The list of found rooms:\n ");
+            outputListInConsole(foundRooms.entrySet());
+            outputSplitLine();
+            bookRoomMenu(reservedFrom, reservedTo, foundRooms);
+            outputDataInConsole("For room reservation please fill out the following form");
+        }
+        return foundRooms;
+    }
+
+    private void bookRoomMenu(Date reservedFrom, Date reservedTo, Map<String, List<Room>> foundRooms) throws IOException {
+        long reserveId = 0L;
+        boolean bookFlaf = true;
+        int numberHotel;
+        Hotel hotelSet = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "hotelSet". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused "hotelSet" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1481)